docs/getting-started/install/dreamhost.md

```diff
@@ -0,0 +1,160 @@
+# Install Dokku on DreamHost Cloud Server with cloud-init
+
+Cloud-init script can be used to automate installation of Dokku on
+Dreamhost (or any other OpenStack-compatible cloud with minimal
+changes).
+
+To create a new server and install Dokku from the openstack command
+line interface:
+
+    $ openstack server create \
```

Comment:
Can you use the backticks for code and highlight this block as shell as shown [here](https://help.github.com/articles/creating-and-highlighting-code-blocks/#syntax-highlighting)?